src/main/java/net/snowflake/client/core/QueryExecDTO.java

```diff
@@ -24,6 +25,8 @@ public class QueryExecDTO {
 
   private Map<String, Object> parameters;
 
+  private QueryContextDTO queryContextDTO;
```

Comment:
/**
 * Optional query context sent to the JDBC driver from the Cloud Service. 
 */

Reply:
add this comment